mdp/test/__init__.py

```diff
@@ -27,7 +27,8 @@ def test(filename=None, seed=None, options='', mod_loc=None):
     # add --assert=reiterp option to work around permissions problem
     # with __pycache__ directory when MDP is installed on a normal
     # user non-writable directory
-    options = "--assert=reinterp "+options
+    # options = "--assert=reinterp "+options
+    options = "-s "+options
```

Comment:
I think here the problem is that you did not merge master on your branch recently? The "assert=reinterp" option to py.test has been removed with the last commit in master: d44efa5 . As that was a revert commit, it creates a conflict. Damn, next time I should simply revert manually and create a new commit...